styles.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* Visuals */
+  background-color: #fefffe;
+}
+
+.container {
+  /* Positioning */
+  width: 100%;
+  height: 100%;
+
+  /* Box Model */
+  margin: 0 auto;
```

Comment:
You set this element to 100% the width of its containing box, so auto margins are not doing you any good here.